src/translator/service.h

```diff
@@ -73,7 +74,7 @@ class Service {
   /// wherein empty memory is passed and internal flow defaults to file-based
   /// model, shortlist loading.
   explicit Service(Ptr<Options> options)
-      : Service(options, AlignedMemory(), AlignedMemory()) {}
+      : Service(options, AlignedMemory(), AlignedMemory(), {}) {}
```

Comment:
I agree with @kpu now, default constructors are okay - get rid of this double tax; Probably a note in docs/comments in the other constructor that `AlignedMemory()` corresponds to empty memory and leads to loading from files supplied through config. 

This is my bad, sorry. 🤦‍♂️